Unit_15/wdio googleCloud/test/pageobjects/gCloud.page.js

```diff
@@ -0,0 +1,7 @@
+module.exports = class gCloudPage {
+  async open(link) {
+    await browser.maximizeWindow();
+    //await browser.fullscreenWindow(); //which one is preferable?
```

Comment:
Answer: Depends on needs, but maximizeWindow() allow to check url behavior on your own.